### ISSUE TITLE
Undersized interface on high-DPI Linux

### ISSUE THREAD

**Issue by nclm** [Aluxian/Messenger-for-Desktop#388]
On a Linux (Ubuntu Gnome) laptop with a high-DPI display, everything is two times too small. Here’s Messenger next to messenger.com in Firefox:

![Too small & Right size](https://cloud.githubusercontent.com/assets/923008/14582718/2ab61d2c-040d-11e6-9a23-c9baab7c8b5a.png)

Also, note the Unknown title in the top bar.

**Comment by efournival** [comment]
You can edit /usr/share/applications (under root) and replace the `Exec=` line by this:
```
Exec=messengerfordesktop --force-device-scale-factor=1.25
```

**Comment by vemek** [comment]
Unfortunately, the I haven't been able to get that argument to work for either `/opt/MessengerForDesktop/messengerfordesktop.desktop` or `/usr/share/applications/messengerfordesktop.desktop`, the two places this file seems to exist after installing on Debian Jessie. However, it does work when executing `messengerfordesktop` from a shell, which is fine for the time being. FWIW, a scaling factor of 2 seems about right for a retina MacBook display.

Please let me know if I can provide any debugging information to help resolve this. In the meantime, I'll try to figure out why changing the desktop shortcut doesn't work on my machine. Thanks for the great wrapper for messenger - makes it vastly more usable on the desktop :)

**Comment by efournival** [comment]
Correct me if I'm wrong but I think that the `.desktop` file is only taken into account when you launch MFD from the Gnome's activities menu. If you are launching it from the shell, you have to edit your `.bashrc` (if using Bash) and add a new alias, for example:
```
alias messengerfordesktop="messengerfordesktop --force-device-scale-factor=2"
```
Regarding your shortcut, I don't exactly know how they work under Gnome but you can use [Arronax](http://www.florian-diesch.de/software/arronax/) to edit the arguments passed to MFD.

**Comment by shihrer** [comment]
Is this project using nwjs?  v0.16+ should automatically handle dpi scaling based on system settings.

**Comment by shihrer** [comment]
What version of nwjs is this project using?  DPI issues shouldn't be a problem with the later versions.  I can't find an exact version when it was fixed, but can confirm v0.16+ handles DPI well.  There shouldn't be any need for command args having to be passed.

**Comment by cg505** [comment]
My laptop has a high DPI display, and I don't have an issue. I use Arch, and I have installed `messengerfordesktop-bin` from the AUR (https://aur.archlinux.org/packages/messengerfordesktop-bin/).
